src/electron/views/results/results-view.tsx

```diff
@@ -115,6 +115,11 @@ export class ResultsView extends React.Component<ResultsViewProps> {
             deps.getDateFromTimestamp,
         );
 
+        const visualHelperSectionProps: VisualHelperSectionProps = {
+            viewModel: screenshotViewModel,
```

Comment:
It feels like `screenshotViewModel` shouldn't be set at this moment because the `visualHelperSection` (in the future) might not be a screenshot. Would we want to refactor this once a non-screenshot `visualHelperSection` exists, or am I misunderstanding something?

Reply:
Yep! I'm going to re-factor the calculation of the view model into the screenshot model. Just wanted this to be a smaller PR.

Reply:
Thanks! Just wanted to make sure I understood right 🙂